### ISSUE TITLE
nested tab icon not show even showIcon=true

### ISSUE THREAD

**Issue by zxccvvv** [react-navigation/react-navigation#3660]
I've read many issue of react-navigation to use vector icon, but still i cant get it 

Why icon not show when I used right `name="name of icon"` but showing question mark when I used wrong `name="name of icon"`

```export const Tab =  TabNavigator({
  Home: {
    screen: Tabs,
    navigationOptions: {
      tabBarLabel: 'Home',
      tabBarIcon: ({ tintColor, focused }) => <Feather containerStyle={{ marginTop: 1 }} name="home" size={25} color={tintColor} />,
      headerTitle: (
        <Image
        style={{
          flex: 1,
          width: 100,
          height: 40,
          resizeMode: 'contain',
          justifyContent: 'center',
          alignItems: 'center',
        }}
        source={require('./google.png')}/>
      ),
    },
  },
  MyPatient: {
    screen: MyPatient,
    navigationOptions: {
      tabBarLabel: 'My Patient',
      tabBarIcon: ({ tintColor }) => <Feather containerStyle={{ marginTop: 1 }} name="people" size={25} color={tintColor} />,
    }
  },
  Login: {
    screen: Login,
  }
},{
  tabBarPosition: 'bottom',
  tabBarOptions: {
    showIcon: true,
    tabStyle: {
      height: 50,
      paddingVertical: 5,
    },
    style: { margin : 1 },
    upperCaseLabel: false,
  },
});
```

and the result is 

![image](https://user-images.githubusercontent.com/19633881/36962152-f98e6938-2080-11e8-9da3-85800157fe91.png)

any idea please?

**Comment by ssssssssssss** [comment]
Are you using RN 0.54?

**Comment by matthamil** [comment]
Hi! Looks like you're missing some info from the [issue template](https://github.com/react-navigation/react-navigation/blob/master/.github/ISSUE_TEMPLATE.md). Can you reproduce this with [Snack](https://snack.expo.io/)? What is your environment (see below)?


| software         | version
| ---------------- | -------
| react-navigation |
| react-native     |
| node             |
| npm or yarn      |

**Comment by mtt87** [comment]
I think there might be some issues with `RN 0.54.0` and tab icons at the moment, they disappeared for me as well and decided to downgrade for the moment.

**Comment by zxccvvv** [comment]
@ssssssssssss @matthamil yes, im using `react native 0.54.0`, and if it might be some issues with `RN 0.54.0` which version of RN should i use?

### My Environment

| software         | version
| ---------------- | -------
| react-navigation | ^1.4.0
| react-native-cli | 2.0.1
| react-native     | 0.54.0
| npm or yarn      | 5.6.0

**Comment by ssssssssssss** [comment]
Downgrade to 0.53.x

**Comment by zxccvvv** [comment]
@ssssssssssss thanks for the suggest, it's work

**Comment by Blutude** [comment]
Will this be fixed for ReactNative 0.54.x?
